deployment/vagrant-development/Vagrantfile

```diff
@@ -9,6 +9,10 @@ check_provider_needs(provider)
 
 Vagrant.configure("2") do |config|
 
+  config.ssh.shell = "bash -c 'BASH_ENV=/etc/profile exec bash'"
+
+  #config.vm.synced_folder "/Users/qetscontractor/.m2/", "/home/vagrant/.m2/",create:false
+
```

Comment:
Can you remove these changes from your pull request?

If you have improvements to Vagrant, please make a separate PR so we can track them and the underlying reasons for them.